gobblin-modules/gobblin-kafka-09/src/test/java/org/apache/gobblin/service/GobblinServiceManagerTest.java

```diff
@@ -181,19 +192,107 @@ public void cleanUp() throws Exception {
   }
 
   @Test
-  public void testCreate() throws Exception {
+  public void testRestart() throws Exception {
+    Map<String, String> flowProperties = Maps.newHashMap();
+    flowProperties.put("param1", "value1");
```

Comment:
Can we move the FlowSpec creation to a separate method? Might even be useful to move it a test utils class that can be reused across different Test classes.